### ISSUE TITLE
Error with "useComponents" option

### ISSUE THREAD

**Issue by KazakovKirill** [nuxt-community/cloudinary-module#9]
Hi!
I'm trying to use vue components and when I added "useComponent" option to module config I getting this error:

**console output:**
```
ERROR  in ./.nuxt/cloudinary/plugin.js

Syntax Error: Unexpected token (6:14)

  4 | const configuration = {
  5 |   cloudName: 'loremipsum',
> 6 |   privateCdn: ,
    |               ^
  7 |   secure: true
  8 | }
  9 | 
```

**nuxt.config.js:**
```
modules: [
    [
      '@nuxtjs/cloudinary',
      {
        cloudName: 'loremipsum',
        useComponent: true,
      },
    ],
  ],
```

**Comment by marcoribeiroweb** [comment]
Hi!
I have the same issue, a temporary fix is to add the `privateCDN` option.
Like this:

**nuxt.config.js:**
`modules: [
    [
      '@nuxtjs/cloudinary',
      {
        cloudName: 'loremipsum',
        privateCDN: false,
        useComponent: true,
      },
    ],
  ],`

Hope it helps! 👍

**Comment by mayashavin** [comment]
Hi, this is fixed in the current master. And will be included in the release this week. Sorry for these inconvenience ☺️

**Comment by mayashavin** [comment]
Fix on v0.0.9
